odoo/addons/base/tests/test_ir_mail_server.py

```diff
@@ -337,6 +337,34 @@ def test_mail_server_send_email_smtp_session(self):
             from_filter='test.com',
         )
 
+    @mute_logger('odoo.models.unlink')
+    def test_mail_server_testing_email_addresses(self):
```

Comment:
"test_mail_server_get_email_addresses"? (Adding "testing" in a test is a bit confusing ^^)

Order alphabetically?